### PR TITLE
Update Dockerfile to build an image

### PR DESCRIPTION
**Pull request by k-hashimoto**
Update Dockerfile to install on local environment.

`civitaspo/redash-qr` dose not exist on dockerhub. So I need to build a docker image. 

- Run an [instruction on README](https://github.com/civitaspo/redash-query-replace#development) on Dockerfile.
- Remove "does something useful" from spec/redash/query/replace_spec.rb.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+RUN apk update && \
+	apk upgrade && \
+	apk add --no-cache git gcc g++ libc-dev linux-headers make
+
+RUN source bin/setup && rake spec && bundle exec rake install
+
+ARG REDASH_URL
+ARG REDASH_API_KEY
+ENV REDASH_URL $REDASH_URL
+ENV REDASH_API_KEY $REDASH_API_KEY
```
Comment (civitaspo) [inline Dockerfile]: These variables must not be defined. If this image are pushed to a public docker registry, any people can get the api key and url from this image.

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,17 @@
 FROM ruby:2.5-alpine
```
Comment (civitaspo) [inline Dockerfile]: I am sorry to upload the Dockerfile that doesn't work. Could you rewrite like the below?

```
FROM ruby:2.5 as builder

WORKDIR /tmp/redash-query-replace
COPY . /tmp/redash-query-replace
RUN gem build redash-query-replace.gemspec

FROM ruby:2.5

WORKDIR /work
COPY --from=builder /tmp/redash-query-replace/redash-query-replace-0.0.1.gem .
RUN gem install redash-query-replace-0.0.1.gem

ENTRYPOINT ["redash-qr"]
```

**Review by civitaspo — CHANGES_REQUESTED** [review]
Please write the commit messages in English.

**Comment by k-hashimoto** [comment]
I'm sorry for quite slow replay. Dockerfile was updated to remove `ARG` and 'ENV'.  `--env-file` option should be used. 

I checked a docker image can be built and cam be used like 
`docker run --rm -v $(pwd)/:/vol civitaspo/redash-qr query --from 'bb' --to 'aa' --id 363875 --env-file=/vol/.env`

 @civitaspo

File: README.md [hunk 3]
```diff
@@ -80,6 +80,18 @@ And then execute:
 Or install it yourself as:
 
     $ gem install redash-query-replace
+	
+### Use Docker	
+
+Build docker image on local
+```
+docker build -t ${USER}/redash-rp --build-arg REDASH_URL=<your redash url> --build-arg REDASH_API_KEY=<your redash api key>
```
Comment (civitaspo) [inline README.md]: Please change this instruction.

File: README.md [hunk 4]
```diff
@@ -80,6 +80,18 @@ And then execute:
...
+```
+
+Run docker image
+```
+docker run --rm -v `pwd`/:/vol ${USER}/redash-rp query --from 'database.table1' --to 'database.table2' --id 5
```
Comment (civitaspo) [inline README.md]: ditto

**Review by civitaspo — CHANGES_REQUESTED** [review]
Modify a bit.

**Comment by k-hashimoto** [comment]
Maybe we doesn't need `Use Docker` section. Because I removed `ARG` and `ENV` from Dockerfile.

**Review by civitaspo — APPROVED** [review]

**Comment by civitaspo** [comment]
thx

**Pull request merged**